doc/api/tls.md

```diff
@@ -918,81 +890,87 @@ added: v0.11.13
-    can be provided either as an array of key strings or as an array of objects
-    in the format `{pem: key, passphrase: passphrase}`. This option is
-    *required* for ciphers that make use of private keys.
-  * `passphrase` {string} A string containing the passphrase for the private key
-    or pfx.
-  * `cert` {string} A string containing the PEM encoded certificate
-  * `ca`{string|string[]|Buffer|Buffer[]} A string, `Buffer`, array of strings,
-    or array of `Buffer`s of trusted certificates in PEM format. If omitted,
-    several well known "root" CAs (like VeriSign) will be used. These are used
-    to authorize connections.
-  * `crl` {string|string[]} Either a string or array of strings of PEM encoded
-    CRLs (Certificate Revocation List).
-  * `ciphers` {string} A string describing the ciphers to use or exclude.
-    Consult
-    <https://www.openssl.org/docs/man1.0.2/apps/ciphers.html#CIPHER-LIST-FORMAT>
-    for details on the format.
-  * `honorCipherOrder` {boolean} If `true`, when a cipher is being selected,
-    the server's preferences will be used instead of the client preferences.
+  * `pfx` {string|Buffer} Optional PFX or PKCS12 encoded private key and
+    certificate chain. `pfx` is an alternative to providing `key` and `cert`
+    individually. PFX is usually encrypted, if it is, `passphrase` will be used
+    to decrypt it.
+  * `key` {string|string[]|Buffer|Buffer[]|Object[]} Optional private keys in
+    PEM format. Single keys will be decrypted with `passphrase` if necessary.
+    Multiple keys, probably using different algorithms, can be provided either
+    as an array of unencrypted key strings or buffers, or an array of objects in
+    the form `{pem: <string|buffer>, passphrase: <string>}`. The object form can
+    only occur in an array, and it _must_ include a passphrase, even if key is
+    not encrypted.
+  * `passphrase` {string} Optional shared passphrase used for a single private
+    key and/or a PFX.
+  * `cert` {string|string[]|Buffer|Buffer[]} Optional cert chains in PEM format.
+    One cert chain should be provided per private key. Each cert chain should
+    consist of the PEM formatted certificate for a provided private `key`,
+    followed by the PEM formatted intermediate certificates (if any), in order,
+    and not including the root CA (the root CA must be pre-known to the peer,
+    see `ca`).  When providing multiple cert chains, they do not have to be in
+    the same order as their private keys in `key`. If the intermediate
+    certificates are not provided, the peer will not be able to validate the
+    certificate, and the handshake will fail.
+  * `ca`{string|string[]|Buffer|Buffer[]} Optional CA certificates to trust.
+    Default is the well-known CAs from Mozilla. When connecting to peers that
+    use certificates issued privately, or self-signed, the private root CA or
+    self-signed certificate must be provided to verify the peer.
```

Comment:
Isn't that how all default values work? The default is used when you don't specify the option. If you specify the option, the default isn't used.

I can spell it out if necessary, but the other `Optional x. Default is y.` don't.

I think the old text used to be confusing, which is why some people thought it could possibly be additive.